source/reference/mongosync.txt

```diff
@@ -296,6 +296,25 @@ error occurs, the ``mongosync`` log may contain the word "error' but
 ``mongosync`` is still able to complete the sync. In the case that a
 sync does not complete, ``mongosync`` writes a fatal log entry. 
 
+UUIDs
+~~~~~
+
+``mongosync`` creates new :abbr:`UUIDs (universally unique identifiers)`
+for collections on the destination cluster. There is no relationship
```

Comment:
mongosync doesn’t “create UUIDs” per se; rather, it creates collections, which have UUIDs.

```suggestion
The collections that ``mongosync`` creates on the destination cluster
have new :abbr:`UUIDs (universally unique identifiers)`. There is no relationship
```

Reply:
Done!